modules/xphoto/src/obstruction_free.cpp

```diff
@@ -0,0 +1,80 @@
+/*M///////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Thank you very much for pointing it out. I will modify the header part accordingly.